conf/Caddyfile.in

```diff
@@ -8,6 +8,20 @@
 	gzip {
 		not /stream
 	}
+
+header /stream {
```

Comment:
Since only one header is added for each path, put these in one line like below instead of opening a block.

```
	header /stream/stream.m3u8 Cache-Control <stuff>
```